### ISSUE TITLE
The app current table view needs reimplmented into the Library button

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/codebooker#48]
The way the app is currently rendering,  the view has a table with books.  This was meant for displaying only when the user clicked on the "Library" button in the left side navigation panel drawer.  This button clearly does not really do anything, nor is there a way to close this view and return to the global scope of the app showing all books added to the database.  

The main app view should show not only books added to the users library but the apps' database(s). of books.  A user could theoretically add a book to their library which has not been added to the app's global scope database yet.

(Note the view currently displayed is actually meant to display when the user clicks on the "Library" button in the drawer, thus showing the books in the users' own library or collection of books showing what they own.)

**Comment by Mayor-Rhymes** [comment]
So, this means that a new component resembling the current library component should be created for the expected table but this table will not contain the user's own library?

**Comment by gbowne1** [comment]
Yeah, what youre seeing now is what should be seen when the user clicks on the Library tab in the left sidenav drawer.

I never really decided what should be seen when the users library is closed. 

Ideally when a user add a book to their library it would add it to the global collection of books so that everyone else csn see them.

Open to design ideas
